armi/reactor/tests/test_components.py

```diff
@@ -1319,6 +1471,13 @@ def test_getVolume(self):
         self.assertAlmostEqual(cur, ref)
 
     def test_updateDims(self):
+        """
+        Test Update dimensions.
+
+        .. test:: Dimensions can be updated
+            :id: T_ARMI_COMP_VOL0
+            :tests: R_ARMI_COMP_VOL
```

Comment:
Again, can you add a test crumb or two for the `test_getArea()` tests in this same file?

This requirement needs BOTH `getVolume()` and `getArea()`.